packages/scripts/config/webpack.config.js

```diff
@@ -302,7 +303,9 @@ const config = {
 					filter: ( filepath ) => {
 						return (
 							process.env.WP_COPY_PHP_FILES_TO_DIST ||
-							RenderPathsPlugin.renderPaths.includes( filepath )
+							RenderPathsPlugin.renderPaths.includes(
+								realpathSync( filepath ).replace( /\\/g, '/' )
+							)
```

Comment:
This is where we decide whether to copy the PHP file containing `render.php`, but it is a bit complicated. Debugging shows that `filepath` contains a backslash and the symlink path is not resolved. On the other hand, `renderPaths` contains a forward slash and the symlink path is resolved.

Therefore, this inconsistency needs to be resolved.

Reply:
Good catch. I agree that we should standardize the path in both places 👍🏻